File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/patient_check_ins_update_patient_check_ins_request.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class PatientCheckInsUpdatePatientCheckInsRequest(TypedDict, total=False):
    CheckInDate: str
    CheckInLocationId: int
    Id: int
    PatientLicenseNumber: str
    RegistrationExpiryDate: str
    RegistrationStartDate: str

File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/PatientCheckInsUpdatePatientCheckInsRequest.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class PatientCheckInsUpdatePatientCheckInsRequest
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("CheckInDate")]
        public string? CheckInDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("CheckInLocationId")]
        public int? CheckInLocationId { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Id")]
        public int? Id { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("PatientLicenseNumber")]
        public string? PatientLicenseNumber { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("RegistrationExpiryDate")]
        public string? RegistrationExpiryDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("RegistrationStartDate")]
        public string? RegistrationStartDate { get; set; }
    }
}
